pkg/loadbalancer/flow/flow.go

```diff
@@ -50,6 +53,7 @@ func New(flow *nspAPI.Flow, nfqueueNumber int, nfqueueFactory networking.NFQueue
 
 func (f *Flow) Update(flow *nspAPI.Flow) error {
```

Comment:
Seems priority update is not supported. Is it intentional?

Reply:
Yes, it is intentional. To update it, it would require to delete the chain and add it again, so it is not possible without traffic interruption.

Reply:
I was more curious about the requirement. But i'd be fine with the approach that one needs to delete a Flow on the operator side first if wishes to change the prio.